packages/caliper-core/test/worker/rate-control/fixedFeedbackRate.js

```diff
@@ -144,5 +144,29 @@ describe('fixedFeedbackRate controller implementation', () => {
             sinon.assert.calledOnce(sleepStub);
             sinon.assert.calledWith(sleepStub, 20000);
         });
+
+        it('should sleep when no successful transactions have occurred', async () => {
+            txnStats.stats.txCounters.totalSubmitted = 5;
+            txnStats.stats.txCounters.totalFinished = 2;
+            txnStats.stats.txCounters.totalSuccessful = 0;
+            controller.generalSleepTime = 1;
+
+            await controller.applyRateControl();
+
+            sinon.assert.calledOnce(sleepStub);
+            sinon.assert.calledWith(sleepStub, 1 * controller.backOffTime);
+        });
+
+        it('should sleep when unfinished transactions are above the threshold', async () => {
+            txnStats.stats.txCounters.totalSubmitted = 44;
+            txnStats.stats.txCounters.totalFinished = 40;
+            controller.unfinishedPerWorker = 1;
```

Comment:
This line is also redundent and will cover it later, however usually I would prefer that this value be defined via the construction of the rate control as the test should be about what a specifically configured rate controller does under different transaction completion state conditions